### PR TITLE
LP Token Database Integration

### PR DESCRIPTION
**Pull request by joehawary**

### CODE REVIEW

File: src/components/crucible/detail/rewards/Subscription.tsx [hunk 1]
```diff
@@ -139,12 +139,13 @@ const Subscription: React.FC<Props> = ({ crucible, subscription }) => {
       stakingTokenPair1 &&
       Object.keys(stakingTokenPair1).length > 0
     ) {
-      const pair0Usd = stakingTokenPair0.value.amountUSD
-        ? formatNumber.token(stakingTokenPair0.value.amountUSD)
+      console.log(stakingTokenPair0.value.amountNumber);
```
Comment (nuurcodes) [inline src/components/crucible/detail/rewards/Subscription.tsx:142]: should we remove console log?

Comment (joehawary) [reply src/components/crucible/detail/rewards/Subscription.tsx:142]: oh dear.. yes definitely

Comment (nuurcodes) [reply src/components/crucible/detail/rewards/Subscription.tsx]: `console.log('thank you')`

**Review by nuurcodes — APPROVED** [review]
Looks good

**Pull request merged**